components/Composition/FAnnouncementCard.jsx

```diff
@@ -11,86 +10,116 @@ import icons from 'themes/icons';
-            imageStyle={styles.image}
-            height={sizes.HEIGHT_150}
-            resizeMode="cover"
-            width="100%"
-          />
-          <FHeading
-            title={data.title}
-            weight={fonts.HEADING_WEIGHT_BOLD}
-            size={fonts.HEADING_MEDIUM}
-            marginBottom={sizes.MARGIN_5}
-            style={{ marginTop: sizes.MARGIN_8 }}
-            ellipsizeMode="tail"
-            numberOfLines={1}
-          />
-          <FHeading
-            title={data.description}
-            marginBottom={sizes.MARGIN_10}
-            color={colors.DARK_GRAY}
-            ellipsizeMode="tail"
-            numberOfLines={2}
-            size={fonts.HEADING_NORMAL}
-            weight={fonts.HEADING_WEIGHT_REGULAR}
-          />
-          <FHeadingWithIcon
-            icon={icons.LOCATION_OUTLINE}
-            iconColor={colors.PRIMARY}
-            title={data.locationName}
-            titleStyle={styles.text}
-            titleColor={colors.DARK_GRAY}
-            numberOfLines={2}
-            iconPlacement={placements.LEFT}
-            iconSize={sizes.ICON_20}
-            titleSize={fonts.HEADING_SMALL}
-            titleWeight={fonts.HEADING_WEIGHT_REGULAR}
-          />
-          <FHeadingWithIcon
-            icon={icons.CALENDAR}
-            iconColor={colors.PRIMARY}
-            title={data.date}
-            titleStyle={styles.text}
-            titleColor={colors.DARK_GRAY}
-            size={fonts.HEADING_MEDIUM}
-            weight={fonts.HEADING_NORMAL}
-            iconPlacement={placements.LEFT}
-            iconSize={sizes.ICON_20}
-            titleSize={fonts.HEADING_SMALL}
-            titleWeight={fonts.HEADING_WEIGHT_REGULAR}
-          />
-        </View>
-      </FCard>
-    </View>
-  </TouchableWithoutFeedback>
-);
+          <View style={{
+            minHeight: height,
+            maxHeight: height,
+          }}
+          >
+            <FImage
+              imagePath=""
+              networkImageUrl={data.photos[0]}
+              imageStyle={styles.image}
+              height={sizes.HEIGHT_150}
+              resizeMode={sizes.COVER}
+              width={sizes.WIDTH_FULL}
+            />
+            <FHeading
+              title={data.title}
+              weight={fonts.HEADING_WEIGHT_BOLD}
+              size={fonts.HEADING_MEDIUM}
+              marginBottom={sizes.MARGIN_5}
+              style={{
+                marginTop: sizes.MARGIN_8,
+                ...titleStyle,
+              }}
+              ellipsizeMode="tail"
+              numberOfLines={1}
+            />
+            <FHeading
+              title={data.description}
+              marginBottom={sizes.MARGIN_10}
+              color={colors.DARK_GRAY}
+              ellipsizeMode="tail"
+              numberOfLines={2}
+              size={fonts.HEADING_NORMAL}
+              weight={fonts.HEADING_WEIGHT_REGULAR}
+              style={{ ...titleStyle }}
```

Comment:
to samo co wyżej. To że FHeading ma propsa style to nie znaczy że trzeba go wykorzystać. On jest w razie w gdybyś chciała dodać jakieś konkretne style dla danego komponentu, który się np wyróżnia lub po prostu czegoś mu brakuje żeby pasował do designu. Nie ma potrzeby przekazywać tego propsa do wszystkich komponentów